class jpeError(Exception):
    pass

class calculationError(jpeError):
    pass
